adam-apis/src/test/scala/org/bdgenomics/adam/apis/java/JavaADAMContextSuite.scala

```diff
@@ -26,19 +26,73 @@ import org.bdgenomics.formats.avro.AlignmentRecord
 
 class JavaADAMContextSuite extends ADAMFunSuite {
 
-  sparkTest("can read a small .SAM file") {
+  sparkTest("can read and write a small .SAM file") {
     val path = copyResource("small.sam")
-    val ctx = new JavaADAMContext(sc)
-    val reads = ctx.loadAlignments(path)
-    assert(reads.jrdd.count() === 20)
+    val aRdd = sc.loadAlignments(path)
+    assert(aRdd.jrdd.count() === 20)
+
+    val newRdd = JavaADAMReadConduit.conduit(aRdd, sc)
```

Comment:
Alas, they can't be inlined because they're written in Java and the test suite is written in Scala. I was unaware that they were public. Perhaps the best route forward is to make them package private?